content/en/_archives/api/configuration-generate.md

```diff
@@ -197,6 +197,13 @@ Interval (in milliseconds) between two render cycles to avoid flooding a potenti
 - **Deprecated!**
 - Use [build.html.minify](/api/configuration-build#html-minify) instead
 
+## nojekyll
+
+- Type: `Boolean`
+- Default: `true`
+
+By default, adding `.nojekyll` file for serving static pages on GitHub Pages. If you want to disable this feature you can set the value to `false`
```

Comment:
```suggestion
Adds a `.nojekyll` file to enable serving `/_nuxt/` when deployed on GitHub Pages. If you want to disable this feature you can set the value to `false`.
```

Reply:
Thank you for your suggestion. I committed it.